content/intro-to-storybook/vue/es/screen.md

```diff
@@ -0,0 +1,209 @@
+      TaskList,
+    },
+  };
+</script>
+```
+
+Luego, podemos crear un contenedor, que nuevamente toma los datos para `PureInboxScreen` en `src/components/InboxScreen.vue`:
+
+```html
+<template>
+  <div>
+    <pure-inbox-screen :error="error" />
+  </div>
+</template>
+
+<script>
+  import PureInboxScreen from './PureInboxScreen';
+  import { mapState } from 'vuex';
+
+  export default {
+    name: 'inbox-screen',
+    components: {
+      PureInboxScreen,
+    },
+    computed: {
+      ...mapState(['error']),
+    },
+  };
+</script>
+```
+
+También cambiamos nuestro componente `App` para que incluya `InboxScreen` (en una aplicación real esto sería manejado por el enrutador pero podemos obviarlo):
+
+```html
+<template>
+  <div id="app">
+    <inbox-screen />
+  </div>
+</template>
+
+<script>
+  import store from './store';
+  import InboxScreen from './components/InboxScreen.vue';
+  export default {
+    name: 'app',
+    store,
+    components: {
+      InboxScreen,
+    },
+  };
+</script>
+<style>
+  @import './index.css';
+</style>
+```
+
+Sin embargo, al intentar mostrar nuestro componente "contenedor" dentro de Storybook las cosas se ponen interesantes.
+
+Como vimos anteriormente, el componente `TaskList` es un **contenedor** que renderiza el componente de presentación `PureTaskList`. Por definición, los componentes contenedores no pueden renderizarse de manera aislada; esperan que se les pase algún contexto o servicio. Esto significa que para mostrar nuestro componente en Storybook, debemos mockearlo (es decir, proporcionar una versión ficticia) del contexto o servicio que requiere.
+
+Al colocar la "Lista de tareas" `TaskList` en Storybook, pudimos esquivar este problema simplemente renderizando la `PureTaskList` y evadiendo el contenedor. Haremos algo similar y renderizaremos la `PureInboxScreen` en Storybook también.
+
+Sin embargo, para la `PureInboxScreen` tenemos un problema porque aunque la `PureInboxScreen` en si misma es presentacional, su hijo, la `TaskList`, no lo es. En cierto sentido la `PureInboxScreen` ha sido contaminada por la "contenedorización". Entonces, cuando configuramos nuestras historias en `src/components/PureInboxScreen.stories.js`:
+
+```javascript
+import PureInboxScreen from './PureInboxScreen.vue';
+export default {
+  title: 'PureInboxScreen',
+};
+
+// inbox screen default state
+export const Default = () => ({
+  components: { PureInboxScreen },
+  template: `<pure-inbox-screen/>`,
+});
+
+// inbox screen error state
+export const error = () => ({
+  components: { PureInboxScreen },
+  template: `<pure-inbox-screen :error="true"/>`,
+});
+```
+
+Vemos que aunque la historia de `error` funciona bien, tenemos un problema en la historia `default`, porque la `TaskList` no tiene una store de Vuex a la que conectarse. (También encontrarás problemas similares cuando intentes probar la `PureInboxScreen` con un test unitario).
+
+![Broken inbox](/intro-to-storybook/broken-inboxscreen.png)
```

Comment:
@IvanSotelo the image is incorrect in this case, you look at it. The error message is for Redux, not vuex. The image should be `broken-inboxscreen-vue.png`